tests/functional/report/monitoring.js

```diff
@@ -0,0 +1,106 @@
+'use strict'; // eslint-disable-line strict
+const http = require('http');
+const assert = require('assert');
+
+describe('Monitoring - getting metrics', () => {
+    const conf = require('../config.json');
+
+    async function query(path, method = 'GET', token = 'report-token-1') {
+        return new Promise(resolve => http.request({
+            method,
+            host: conf.ipAddress,
+            path,
+            port: 8000,
+            headers: { 'x-scal-report-token': token },
+        }, () => resolve()).end());
+    }
+
+    async function getMetrics() {
+        return new Promise(resolve => {
+            http.get({ host: conf.ipAddress, path: '/metrics', port: 8002 }, res => {
+                assert.strictEqual(res.statusCode, 200);
+
+                let body = '';
+                res.on('data', chunk => { body += chunk; });
+                res.on('end', () => resolve(body));
+            });
+        });
+    }
+
+    function parseMetric(metrics, name, labels) {
+        const labelsString = Object.entries(labels !== undefined ? labels : {
+            method: 'GET', code: '200', route: 'healthcheck',
+        }).map(e => `${e[0]}="${e[1]}"`).join(',');
```

Comment:
```suggestion
        }).map(e => e.join('=')).join(',');
```

Reply:
This does not work, we need to add the double-quotes in the string